### PR TITLE
Bug card empty reference

### PR DESCRIPTION
**Pull request by LouisDelbosc**
#402 

https://user-images.githubusercontent.com/11460746/215151021-3723a324-d50e-4b11-a88f-b799ff19703d.mov

Thanks to @dsagal to have found the fix !

I made some modification on the `Favorite_Films.grist` file for the test to be easier.

### CODE REVIEW

**Comment by vviers** [comment]
Hi there ! 👋🏼 

Do you think that we can expect this bugfix to be a part of next week's release ? No rush but we are going live with a project that will benefit from this being fixed and we are trying to figure out whether we need to warn our users about this :)

Thank you so much @LouisDelbosc for the fix and @dsagal for you help in the issue :)

File: test/nbrowser/ReferenceColumns.ts [hunk 1]
```diff
@@ -147,6 +147,23 @@ describe('ReferenceColumns', function() {
         ]
       );
     });
+
+    it('should have linked card for friends', async () => {
```
Comment (berhalak) [inline test/nbrowser/ReferenceColumns.ts:151]: Can you use another document for this test, or create a new one? It is havely used in tests that are not yet part of the grist-core and this additional row breaks them.

**Comment by berhalak** [comment]
> Hi there ! 👋🏼
> 
> Do you think that we can expect this bugfix to be a part of next week's release ? No rush but we are going live with a project that will benefit from this being fixed and we are trying to figure out whether we need to warn our users about this :)

Hi, sorry about the delay. It will be a part of the release after the next one (on Feb. 13th).

**Comment by berhalak** [comment]
Thanks for the update @LouisDelbosc. This fix works well, but I have some doubts about using this 'new' identifier. It is also exposed in custom widgets as a special event `onNewRecord`, and I'm not quite sure that this is a good idea to reuse it as a `no-data` state. I feel that by fixing the card view we will be breaking custom widgets. I need some more time to think about it.

**Review by berhalak — APPROVED** [review]
Thanks, @LouisDelbosc. I tested custom widgets, and this doesn't break them, and actually it looks consistent with other linked behavior.

**Pull request merged**